<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-Q8UQAUDS" ignored="0" name="edge" tests="1" failures="1" timestamp="2023-05-22T08:52:40 IST" time="12.895" errors="0">
  <testcase name="Test" time="12.796" classname="testNG.Parallel_Test">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at testNG.Parallel_Test.Test(Parallel_Test.java:77)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Test -->
</testsuite> <!-- edge -->
